### PR TITLE
Label some default entity collision/pyhsics handlers and everything i…

### PR DESCRIPTION
**Pull request by daid**
…n the beetle spawner code except for some minor jumps

### CODE REVIEW

File: src/code/entities/bank3.asm [hunk 1]
```diff
@@ -8124,7 +8124,7 @@ Data_003_788B::
     db   $01, $FF, $00, $00, $01, $01, $FF, $FF
 
 ; Another function for entity physics
```
Comment (kemenaran) [inline src/code/entities/bank3.asm:8126]:
```suggestion
; Handle physics between an entity and the Background / objects.
; This includes not being able to clip walls, falling in pits and stuff.
```

File: src/code/entities/bank3.asm [hunk 2]
```diff
@@ -6182,7 +6182,7 @@ jr_003_6E0E:
     scf                                           ; $6E26: $37
     ret                                           ; $6E27: $C9
 
-func_003_6E28::
+DefaultEnemyDamageCollisionHandler::
```
Comment (kemenaran) [inline src/code/entities/bank3.asm:6185]:
```suggestion
; Handle collision and damages between Link and the active entity.
; This includes the entity being damaged by sword, the position of Link being modified, etc.
DefaultEnemyDamageCollisionHandler::
```

**Review by kemenaran — APPROVED** [review]
Yay, thanks for the naming.

I added two comments describing the general purpose of each functions, based on a very fast overview of the code, and what you wrote on Discord; tell me if it matches what you saw.

**Comment by daid** [comment]
I think that's right. At least, better then what it was. These functions are quite large and complex and behavior does depend on entity flags and types.

**Comment by kemenaran** [comment]
Oops, I forgot about this one. Rebased and merged.

**Pull request merged**